docs/sources/cli.md

```diff
@@ -18,12 +18,12 @@ Grafana CLI is a small executable that is bundled with Grafana server. It can be
 To list all commands and options:
 
 ```
-grafana-cli -h
+grafana cli -h
 ```
 
 ## Invoking Grafana CLI
 
-To invoke Grafana CLI, add the path to the grafana binaries in your `PATH` environment variable. Alternately, if your current directory is the `bin` directory, use `./grafana-cli`. Otherwise, you can specify full path to the CLI. For example, on Linux `/usr/share/grafana/bin/grafana-cli` and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana-cli.exe`.
+To invoke Grafana CLI, add the path to the grafana binaries in your `PATH` environment variable. Alternately, if your current directory is the `bin` directory, use `./grafana cli`. Otherwise, you can specify full path to the CLI. For example, on Linux `/usr/share/grafana/bin/grafana cli` and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana cli.exe`.
```

Comment:
`and on Windows `C:\Program Files\GrafanaLabs\grafana\bin\grafana cli.exe`.

this seems broken - empty space before exe?

Reply:
ah good catch.

Reply:
fixed